easybuild/easyconfigs/m/MultiQC/MultiQC-1.7-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,107 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Adam Huffman
+# The Francis Crick Institute
+# Elements derived from work by Pablo Escobar
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'PythonBundle'
+
+name = 'MultiQC'
+version = '1.7'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://multiqc.info'
+description = """Aggregate results from bioinformatics analyses across many samples into a single 
+ report.
+
+ MultiQC searches a given directory for analysis logs and compiles a HTML report. It's a general
+ use tool, perfect for summarising the output from numerous bioinformatics tools."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('matplotlib', '3.0.0', versionsuffix),
+    ('PyYAML', '3.13', versionsuffix),
+]
+
+use_pip = True
+
```

Comment:
Please also add `download_dep_fail = True`

Reply:
@vanzod `download_dep_fail = True` is already the default in `PythonBundle` easyblock.

Reply:
Confirmed. I do understand the confusion though, best way out is to have a check in Travis for this and start relying on it...